src/ILVerification/src/Verifier.cs

```diff
@@ -23,15 +23,19 @@ public class Verifier
             new Lazy<ResourceManager>(() => new ResourceManager("FxResources.ILVerification.SR", typeof(Verifier).GetTypeInfo().Assembly));
 
         private ILVerifyTypeSystemContext _typeSystemContext;
+        private VerifierOptions _verifierOptions;
 
-        public Verifier(IResolver resolver)
+        public Verifier(IResolver resolver, VerifierOptions verifierOptions) : this(new ILVerifyTypeSystemContext(resolver), verifierOptions) { }
```

Comment:
We can also have a convenience constructor `public Verifier(IResolver resolver)` that assumes the default options.

Reply:
ok

Reply:
done thank's for your time Jan!